composer.json

```diff
@@ -1,4 +1,11 @@
 {
+    "autoload": {
+        "files": ["fatfree/lib/base.php"],
+        "psr-4": {
+            "Models\\":"fatfree/app/models",
+            "Controller\\":"fatfree/app/controllers"
```

Comment:
This may need to be `"Models\\":"fatfree/app/Controllers"`.  Note the capital "C" in controllers.